src/msc/msc.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <IsTool>true</IsTool>
     <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
+    <ProjectName>Minsk</ProjectName>
```

Comment:
What does `<ProjectName>` control?

Reply:
It's just a meta, but this is used here like this... Publish to a common directory.

https://github.com/terrajobst/minsk/blob/44be9e089142fb1b6ac113f4bb02b92563fbfd70/src/Directory.Build.targets#L19